packages/auth-kit/example/package.json

```diff
@@ -13,7 +13,8 @@
     "@emotion/styled": "^11.10.5",
     "@mui/material": "^5.11.7",
     "@safe-global/safe-react-components": "^2.0.1",
-    "ethers": "^6.0.3",
+    "@toruslabs/ethereum-controllers": "^4.5.2",
+    "ethers": "^5.7.2",
```

Comment:
Why this rollback?

Reply:
I need help stuck in finding my digital identity